package-lock.json

```diff
@@ -11,7 +11,7 @@
       "dependencies": {
         "@contentful/rich-text-types": "^16.3.0",
         "@types/json-patch": "0.0.30",
-        "axios": "^1.6.2",
+        "axios": "^1.6.7",
```

Comment:
This line should be unchanged, right?

Reply:
You are right, I forgot to revert this line.